lib/models/updateAction.js

```diff
@@ -92,6 +92,9 @@ function buildUpdateOptions(action, event) {
         id: myutils.expandVar(action.parameters.id, event),
```

Comment:
CNR entry corresponding to the new feature missing.

In addition, the new parameter should be described (and shown in the example) in documentation at https://github.com/telefonicaid/perseo-fe/blob/master/documentation/plain_rules.md#update-attribute-action

(I understand that although the title of the PR mentions "cards", i.e. VR API, the parameter is also valid in the plain rule API. I may be wrong)